modules/tinymce/src/core/test/ts/browser/init/InitEditorNoThemeInlineTest.ts

```diff
@@ -26,13 +26,17 @@ describe('browser.tinymce.core.init.InitEditorNoThemeInlineTest', () => {
     const editor = hook.editor();
     const body = SugarBody.body();
     const targetElement = SelectorFind.descendant(body, '#' + editor.id).getOrDie('No elm');
-    const nextElement = Traverse.nextSibling(targetElement);
+    const nextElement = Traverse.nextSibling(targetElement).getOrDie('Should be an element after the target');
 
     // TODO FIXME this seems like an odd exception
     assert.isNull(editor.getContainer(), 'Should be null since inline without a theme does not set editorContainer');
     Assertions.assertDomEq('Should be expected editor body element', targetElement, TinyDom.body(editor));
     Assertions.assertDomEq('Should be expected editor target element', targetElement, TinyDom.targetElement(editor));
     Assertions.assertDomEq('Editor.contentAreaContainer should equal target element', targetElement, TinyDom.contentAreaContainer(editor));
-    assert.isTrue(nextElement.isNone(), 'Should be no element after target');
+    // The only element that should be after the target is the TableResizeHandler TableWire div element
+    assert.lengthOf(Traverse.nextSiblings(targetElement), 1, 'Should only be one element after target');
+    // test with 5px
+    const resizeDivStructure = ApproxStructure.fromHtml('<div style="position: static; height: 0px; width: 0px; padding: 0px; margin: 0px; border: 0px;"></div>');
```

Comment:
I guess this was part of the original code. But why is a static hidden element needed to be added to the dom outside the editor element? If it's a resize shim shouldn't it be position absolute or fixed any only be added while resizing not always?

Reply:
I am not sure of the exact answer for this. It is probably a good thing to look at in more detail in the future though to see if it can be improved in any way

Reply:
Snooker's table resizing concepts came from TBIO which didn't support bogus elements. To hide resize handles from the content APIs we had to put them outside the body.

For inline editing that naturally means putting them after the inline content element. I don't know why this was done instead of putting the handles in the UI container. Now that resize handles are part of core perhaps this can be investigated (although then we have core assuming our silver theme exists, which is a bad assumption).

The things inside this container div are position absolute as you'd expect. But their container is a nice single div that can ensure cleaning up the handles is just a single DOM remove call.